vms/platformvm/service.go

```diff
@@ -1290,7 +1290,7 @@ func (service *Service) IssueTx(_ *http.Request, args *IssueTxArgs, response *Is
 		service.vm.unissuedAtomicTxs = append(service.vm.unissuedAtomicTxs, tx)
 		response.TxID = tx.ID()
 	default:
-		return errors.New("Could not parse given tx. Must be a TimedTx, DecisionTx, or AtomicTx")
+		return errors.New("Could not parse given tx. Must not be a TimedTx, DecisionTx, or AtomicTx")
```

Comment:
This error message seems to be confusing. Can we make it clear that the provided transaction has to be either a `TimedTx`, `DecisionTx`, or an `AtomicTx`?

"Could not parse given tx. Provided tx needs to be a TimedTx, DecisionTx, or AtomicTx"

Reply:
Updated